addons/van_jsx/src/type.d.ts

```diff
@@ -1,16 +1,23 @@
+import { State, ChildDom } from "vanjs-core";
+
+type VanElement = Element | ChildDom;
```

Comment:
Do you need VanElement? I think ChildDom is a super type of Element (ChildDom > Node > Element).

Reply:
Right, I was in a hurry and didn't consider that, will resolve, just not right now.

Reply:
> Do you need VanElement? I think ChildDom is a super type of Element (ChildDom > Node > Element).

In that case, should the whole ComponentChildren type be just replaced with ChildDom @cqh963852 @Tao-VanJS ?

Reply:
I think `ComponentChildren` should be replaced with `ChildDom`. Technically is `ComponentChildren` a super type of `ChildDom` because the return type of `FunctionChild<P>` is broader than `BindingFunc` defined in VanJS. But this is probably not what we want. A function that returns another function is not a valid binding function in VanJS anyway.